exec-core/src/main/java/com/capitalone/dashboard/exec/model/CollectorType.java

```diff
@@ -16,3 +16,4 @@ public static CollectorType fromString(String value) {
 		throw new IllegalArgumentException(value + " is not a CollectorType");
 	}
 }
+
```

Comment:
Remove this line addition it is not needed

Reply:
This code is not updated by me, please cross verify once again.